lib/src/cmaf/ham/mapper/mpd/utilsHamToMpd.ts

```diff
@@ -0,0 +1,35 @@
+import type { AudioTrack, Track } from '../../types/model';
+
+const VIDEO_SAMPLE_RATE = 90000;
+const TEXT_SAMPLE_RATE = 1000;
+
+/**
+ * This function tries to recreate the timescale value.
+ *
+ * This value is not stored on the ham object, so it is not possible (for now)
+ * to get the original one.
+ *
+ * Just the audio tracks have this value stored on the `sampleRate` key.
+ *
+ * Using 90000 as default for video since it is divisible by 24, 25 and 30
+ *
+ * Using 1000 as default for text
```

Comment:
```suggestion
// Using 90000 as default for video since it is divisible by 24, 25 and 30
const VIDEO_SAMPLE_RATE = 90000;
// Using 1000 as default for text
const TEXT_SAMPLE_RATE = 1000;

/**
 * This function tries to recreate the timescale value.
 *
 * This value is not stored on the ham object, so it is not possible (for now)
 * to get the original one.
 *
 * Just the audio tracks have this value stored on the `sampleRate` key.
 *
 * Using defaults for video and text.
```